tests/conftest.py

```diff
@@ -1,18 +1,67 @@
 """Define global fixtures."""
 
-from os.path import join
-
-import pytest
+from os.path import abspath, dirname, join
+from pickle import load as _load
 
 from cobra import Metabolite, Model, Reaction
-from cobra.test import create_test_model, data_dir
+from cobra.io import read_sbml_model
 from cobra.util import solver as sutil
 
 
 try:
-    from cPickle import load as _load
+    import pytest
+    import pytest_benchmark
 except ImportError:
-    from pickle import load as _load
+    pytest = None
+
+
+cobra_directory = abspath(join(dirname(abspath(__file__)), ".."))
+cobra_location = abspath(join(cobra_directory, "src"))
+data_dir = join(cobra_directory, "tests", "data", "")
+
+
+def create_test_model(model_name="salmonella"):
+    """Return a cobra model for testing.
+
+    model_name: str
+        One of 'ecoli', 'textbook', or 'salmonella', or the
+        path to a pickled cobra.Model
+
+    """
+    if model_name == "ecoli":
+        ecoli_sbml = join(data_dir, "iJO1366.xml.gz")
```

Comment:
With `data_dir` being a `Path` you might have to change these to:

```suggestion
        ecoli_sbml = str(data_dir / "iJO1366.xml.gz")
```